tests/ha/ha_cluster_init.pm

```diff
@@ -26,6 +27,16 @@ sub run {
     my $unicast_opt   = get_var("HA_UNICAST") ? '-u' : '';
     my $quorum_policy = 'stop';
     my $fencing_opt   = "-s $sbd_device";
+    my $qdevice_opt   = '';
+
+    # Qdevice configuration
+    if (get_var('QDEVICE')) {
+        zypper_call 'in corosync-qdevice';
+        my $qnet_node_host = choose_node(3);
+        my $qnet_node_ip   = get_ip($qnet_node_host);
+        $qdevice_opt = "--qnetd-hostname=$qnet_node_ip";
```

Comment:
As `$qnet_node_ip` is only used here, maybe it's better to write:
`$qdevice_opt = "--qnetd-hostname=" . get_ip($qnet_node_host);`